Core/GDCore/Project/EventsFunctionsExtension.h

```diff
@@ -165,6 +195,24 @@ class GD_CORE_API EventsFunctionsExtension : public EventsFunctionsContainer {
    */
   void Init(const gd::EventsFunctionsExtension& other);
 
+  void SerializeDependencyTo(const gd::DependencyMetadata& dependency,
```

Comment:
I am not sure if this belongs here. My thought process for putting this here instead of the dependency metadata class directly is that dependency metadata shouldn't be serialized in it's other usage (in the extension) so it should be here as this is the only case where there is a need to serialize it.

Reply:
Seems reasonable 👍